images/Dockerfile

```diff
@@ -10,7 +10,7 @@ WORKDIR /usr/src/sriov-network-device-plugin
 RUN make clean && \
     make build
 
-FROM golang:1.21-alpine3.19 as ddp-builder
+FROM golang:1.20-alpine3.16 as ddp-builder
```

Comment:
I think a better idea will be to have a different container for the ddp tool we want the go code to be on 1.21

Reply:
I propose to address this in a separate PR, so that we unblock the image build first

Reply:
The go code is built with golang 1.21, ddp uses a different layer

Reply:
@SchSeba can you please open an issue on this ?

Reply:
oh it was already done sorry I miss it...